src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -1433,9 +1433,7 @@ uct_rc_mlx5_iface_common_poll_rx(uct_rc_mlx5_iface_common_t *iface,
 
     cqe = uct_rc_mlx5_iface_poll_rx_cq(iface, poll_flags);
     if (cqe == NULL) {
-        /* If no CQE - post receives */
-        count = 0;
-        goto done;
+        return 0;
```

Comment:
why we don't need to post receives anymore?

Reply:
I thinks the comment was misleading, we post it when handle some CQEs, otherwise we have nothing to post

Reply:
ok, it makes sense to me, since no CQEs were consumed. I was confused by the comment.
wdyt about adding:
```
ucs_assertv(iface->super.rx.srq.available < max_batch, "RX srq available (%u), max_batch (%u)",
            iface->super.rx.srq.available, max_batch);
```

Reply:
ok

Reply:
if we get a bunch of receive errors (which we ignore, and cqe is NULL) still need to post receives

Reply:
yes, did it already but have not pushed yet. Still see some error even when do that